packages/location-management/src/ducks/tests/fixtures.ts

```diff
@@ -0,0 +1,83 @@
+import { LocationTag } from '../location-tags';
+import { LocationUnit, LocationUnitStatus } from '../location-units';
+/* eslint-disable @typescript-eslint/camelcase */
+
+export const locationTag1: LocationTag = {
```

Comment:
seems like the `LocationTag` in defined in the reducer module is not complete. IMO, the interface should target to represent the incoming payload as accurately as possible.

Reply:
@p-netm Sorry, I don't get you

Reply:
I mean the interface's properties do not match those in these objects.

Reply:
@p-netm Ok...I presume you mean `id: string | number` which have changed to `id: number`

Reply:
oooh I see, there are 2 interfaces both called `LocationTags` my bad. 
then nitpick: maybe we should change the one of them to something else  I am thinking the one in `location-units` can read `LocationUnitsTags` or sth

Reply:
@p-netm I have changed it to `LocationUnitTag`